[vtk] Restore IO import error forwarding

Feed through the proper observers as opposed to raising globally.

--- IO/Import/vtkGLTFImporter.cxx
+++ IO/Import/vtkGLTFImporter.cxx
@@ -417,6 +417,8 @@ int vtkGLTFImporter::ImportBegin()
   vtkNew<vtkEventForwarderCommand> forwarder;
   forwarder->SetTarget(this);
   this->Loader->AddObserver(vtkCommand::ProgressEvent, forwarder);
+  this->Loader->AddObserver(vtkCommand::WarningEvent, forwarder);
+  this->Loader->AddObserver(vtkCommand::ErrorEvent, forwarder);

   // Check extension
   std::vector<char> glbBuffer;
